monai/networks/nets/vit.py

```diff
@@ -43,6 +43,7 @@ def __init__(
         num_classes: int = 2,
         dropout_rate: float = 0.0,
         spatial_dims: int = 3,
+        acti="Tanh",
```

Comment:
Not necessary to make this change, but perhaps using the term 'post_activation' might be more insightful to the user. Upto you. :)